control_protocol.md

```diff
@@ -330,17 +330,490 @@ A Component MUST also offer a list of all possibly callable methods in accordanc
 For such a RPC message, the first content frame MUST consist in a JSON-RPC compatible content, for example a single request object or a batch of request objects.
 
 
+### List of methods
+
+List of methods defined by LECO.
+You MAY implement the optional methods of this list and you MUST implement the methods obligatory for your type of Component.
+All methods implemented in a Component MUST adhere to this list or MUST have a name different to any method in this list.
+
+
+#### Component
+
+Any Component, i.e. any participant in the LECO protocol, MUST offer the following methods.
+
+::: json
+{
+    "openrpc": "1.2.6",
+    "info": {
+        "title": "Component",
+        "version": "0.1.0"
+    },
+    "methods": [
+        {
+            "name": "pong",
+            "description": "Respond to a ping.",
+            "params": [],
+            "result": {
+                "name": "result",
+                "schema": {
+                    "type": "null"
+                },
+                "required": true
+            }
+        }
+    ],
+    "components": {
+        "schemas": {}
```

Comment:
what is this for?

Reply:
To hold more information. According to the spec it is not required, so I remove it.